exercises/exercisesRoutes.js

```diff
@@ -0,0 +1,27 @@
+const router = require('express').Router();
+const db = require('./exercise-helpers');
+
+router.get('/', async (req, res) => {
+  try {
+    const exercises = await db.findExercises();
+    res.status(200).json(exercises);
+  } catch (error) {
+    res.status(500).json({ Error: 'Internal Server Error' });
+  }
+});
+
+router.get('/:id', async (req, res) => {
```

Comment:
If the Pull Request is to view all exercises, this endpoint is meant to get one exercise by id should not be included. I think Matt is already working on this.